### DISCUSSION TITLE
How to change the intermediate certificate and key that step-ca uses?

### DISCUSSION THREAD

**Discussion by Mattz0r** [smallstep/certificates#1135] [Q&A] [answered]
Hello there,

I've completed the step ca init, which has generated a certificate for the step ca web interface, however I have since performed the steps to extend our internal CA, by changing the intermediate to be signed by our internal root.

My issue is, that when I access step-ca via the url, it's using a different signed certificate... how do I get it to use one that's been signed against the new root/intermediate?

Thanks in advance, sorry if it's already been covered, I had a quick look but couldn't find anything.

**Comment by Mattz0r** [comment]
This can be closed, I figured it out :)

**Reply by maraino** [reply]
Can you provide an explanation of what the issue was? And just mark that response as the answer.

**Comment by Mattz0r** [comment, marked as answer]
The issue was that I hadn't correctly replaced the intermediate_ca.crt and key file.

Once I had ensured the correct files were in place and restarted the step-ca daemon. Everything worked as expected.
